### PR TITLE
add `Banner` Component

### PR DESCRIPTION
**Pull request by jaf107**
resolves : https://github.com/optimizely-axiom/optiaxiom/issues/337

### CODE REVIEW

File: packages/react/src/banner-description/BannerDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Box } from "../box";
+import { Flex } from "../flex";
+
+type AlertDescriptionProps = ComponentPropsWithRef<typeof Box>;
```
Comment (toha993) [inline packages/react/src/banner-description/BannerDescription.tsx:6]: Lets rename props to BannerDescriptionProps
```suggestion
type BannerDescriptionProps = ComponentPropsWithRef<typeof Box>;
```

File: packages/react/src/banner-description/BannerDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export const BannerDescription = forwardRef<
+  HTMLDivElement,
+  AlertDescriptionProps
```
Comment (toha993) [inline packages/react/src/banner-description/BannerDescription.tsx:10]:
```suggestion
  BannerDescriptionProps
```

File: packages/react/src/banner/Banner.css.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        backgroundColor: theme.colors["brand.200"],
+      }),
+      neutral: style({
+        backgroundColor: theme.colors["bg.neutral.solid"],
+      }),
+      success: style({
+        backgroundColor: theme.colors["green.200"],
+      }),
+      warning: style({
+        backgroundColor: theme.colors["yellow.200"],
```
Comment (toha993) [inline packages/react/src/banner/Banner.css.ts:30]: Let's use sprinkles.

File: apps/storybook/src/feedback/Banner.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          </Text>
+        </BannerDescription>
+      </>
+    ),
+    colorScheme: "danger",
+    onClose: () => {},
+  },
+  parameters: {
+    layout: "padded",
+  },
```
Comment (toha993) [inline apps/storybook/src/feedback/Banner.stories.tsx:39]: Why do we need it?

File: apps/storybook/src/feedback/Banner.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+          <Text>
+            <Link colorScheme="neutral" href="data:,">
+              Sync now
+            </Link>
+          </Text>
+        </BannerDescription>
+      </>
+    ),
+    colorScheme: "danger",
```
Comment (toha993) [inline apps/storybook/src/feedback/Banner.stories.tsx:34]: Lets not change default colorScheme in Basic story

File: apps/storybook/src/feedback/Banner.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+          that requires careful consideration.
+        </BannerDescription>
+      </>
+    ),
+    onClose: action("close"),
+  },
+  render: (args) => (
+    <Flex>
+      <Banner {...args} />
+      <Banner {...args} />
```
Comment (toha993) [inline apps/storybook/src/feedback/Banner.stories.tsx:58]: Is it intentional to showcase both same or we missed the colorScheme to add?

File: apps/storybook/src/feedback/Banner.stories.tsx (hunk 4 above)
Comment (jaf107) [reply apps/storybook/src/feedback/Banner.stories.tsx:39]: to show the banner at the top of the screen in a banner form

Comment (toha993) [reply apps/storybook/src/feedback/Banner.stories.tsx:38]: Then we need to add it in the `Meta`, so every story has the same behaviour.

export default {
  component: Banner,
  parameters: {
    layout: "padded",
  },
} as Meta<typeof Banner>;

Comment (jaf107) [reply apps/storybook/src/feedback/Banner.stories.tsx:38]: These makes a lot more sense. Thanks bro!

**Review by radnan — APPROVED** [review]

**Pull request merged**